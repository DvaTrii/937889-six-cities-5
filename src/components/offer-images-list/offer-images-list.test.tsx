import * as React from "react";
import * as renderer from "react-test-renderer";
import OfferImagesList from "./offer-images-list";

it(`Should OfferImagesList render properly`, () => {
  const pictures: Array<string> = [
    `#`, `#`, `#`, `#`,
  ];


  const tree = renderer
    .create(
        <OfferImagesList
          pictures={pictures}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
