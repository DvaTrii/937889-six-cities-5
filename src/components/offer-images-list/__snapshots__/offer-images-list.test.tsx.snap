// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferImagesList render properly 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="#"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="#"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="#"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="#"
      />
    </div>
  </div>
</div>
`;
